src/controllers/controller.bar.js

```diff
@@ -219,6 +226,11 @@ module.exports = function(Chart) {
 			var custom = rectangle.custom || {};
 			var rectangleOptions = chart.options.elements.rectangle;
 
+			//  float-bar support, if y arguments are array lets override rectangles styles, assigning no skippingBorder
+			if (helpers.isArray(dataset.data[index])) {
+				custom.borderSkipped = null;
```

Comment:
So does that means we will enforce `borderSkipped` in case of float-bar? I'm not sure that's a good idea.

Reply:
yes, in fact we enforce this, cause float-bar will have all 4 sides required to be drawn, now sure if need to add smth else there. Or your idea to check the stacking here and make border skipped when 2 float-bars are stacked and one border should be eliminated. In my opinion in this case we will have 2 borders between 2 float-bars while stacking but thats better for vizualization purpose.